stdlib/mexpr/utesttrans.mc

```diff
@@ -94,45 +182,66 @@ let int2stringName = optionGetOrElse
   (lam. error "Expected int2string to be defined")
   (findName "int2string" utestRunner)
 
+let float2stringName = optionGetOrElse
+  (lam. error "Expected float2string to be defined")
+  (findName "float2string" utestRunner)
+
 let withUtestRunner = lam term.
   bind_ utestRunner term
 
 recursive let _printFunc = use MExprAst in
   lam ty.
-  match ty with TyInt {} then
+  match ty with TyInt _ then
     nvar_ int2stringName
-  else match ty with TyBool {} then
+  else match ty with TyFloat _ then
+    nvar_ float2stringName
+  else match ty with TyBool _ then
     ulam_ "b" (if_ (var_ "b") (str_ "true") (str_ "false"))
+  else match ty with TyChar _ then
+    ulam_ "c" (seq_ [char_ '\'', var_ "c", char_ '\''])
```

Comment:
At some point we should escape special chars (but it can wait I think)

Reply:
I made a fix so that the OCaml pretty-printer should now escape special characters like these, so I think this should not be a problem.